### DISCUSSION TITLE
On V400 require 3.9 python version or later

### DISCUSSION THREAD

**Discussion by Skatedan** [Donkie/Spoolman#313] [General]
Hey there, 
I have a problem trying to install spoolman on my v400 that have 3.8 python version.
I tried to install 3.9 required, but it destroy all my klipper/moonracker system.
How can I install spoolman whitout ruining the system?
Thank you!

**Comment by Donkie** [comment]
You either upgrade your python or install it using the docker method. Python 3.8 is very old and should not be used anymore.

**Reply by Skatedan** [reply]
<img width="816" alt="Screenshot 2024-02-12 at 13 44 08" src="https://github.com/Donkie/Spoolman/assets/52548863/768c380a-ae64-44c5-ba03-0c662b591c83">
I tried, but following guide it doesn't works :/

**Reply by Skatedan** [reply]
<img width="1680" alt="Screenshot 2024-02-12 at 14 00 33" src="https://github.com/Donkie/Spoolman/assets/52548863/95e09689-02da-4df7-81d0-6c1726de9e66">
